src/Microsoft.DotNet.ProjectModel/ProjectReader.cs

```diff
@@ -546,6 +551,29 @@ private static CommonCompilerOptions GetCompilationOptions(JObject rawObject, Pr
                 project.AnalyzerOptions = analyzerOptions;
             }
 
+            IncludeFilesResolver embed = null;
+            var embedOption = rawOptions.Value<JToken>("embed");
+            if (embedOption != null)
+            {
+                embed = new IncludeFilesResolver(
+                    project.ProjectDirectory,
+                    "embed",
+                    rawOptions,
+                    ProjectFilesCollection.DefaultResourcesBuiltInPatterns,
+                    ProjectFilesCollection.DefaultBuiltInExcludePatterns);
```

Comment:
all the information needed to create an `IncludeFilesResolver` can be saved without building this resolver at this moment.